experimental/lib/Transforms/LSQSizing/LSQSizingSupport.cpp

```diff
@@ -41,8 +41,20 @@ using namespace dynamatic::experimental::lsqsizing;
 static int extractNodeLatency(mlir::Operation *op, TimingDatabase timingDB) {
   double latency = 0;
 
-  if (!failed(timingDB.getLatency(op, SignalType::DATA, latency)))
+  if (!failed(timingDB.getLatency(op, SignalType::DATA, latency))) {
+
+    // TODO This is a temporary hack to fix the latency of the LoadOp when its
+    // connected to an LSQ (Necessary since there is now the same loadop for MC
+    // and LSQ loads). When the TimingDB gets reworked, this can be removed
+    if (isa<handshake::LoadOp>(op)) {
+      for (mlir::Operation *destOp : op->getUsers()) {
+        if (isa<handshake::LSQOp>(destOp)) {
+          return latency + 3;
+        }
+      }
+    }
```

Comment:
> I have adapted the code to work with this new change. For the LSQLoad latency, I added a check that adds +3 latency (hardcoded), as suggested in the comment from the other pull request.

Sorry! My previous message was not so clear. 

**You don't need to add this latency here!** Lucas already handled the +3 cycle latency in the getLatency method:
https://github.com/EPFL-LAP/dynamatic/blob/f258670bbefca8fdaa12c18bff2c19ca2f93dfdc/lib/Support/TimingModels.cpp#L114-L141